policyengine_us/reforms/scott_budow/end_entrenched_poverty/end_entrenched_poverty.py

```diff
@@ -0,0 +1,71 @@
+from policyengine_us.model_api import *
+
+
+def create_end_entrenched_poverty_credit() -> Reform:
+    class end_entrenched_poverty_credit(Variable):
+        value_type = float
+        entity = TaxUnit
+        label = "Scott Budow's End Entrenched Poverty Credit"
+        unit = USD
+        definition_period = YEAR
+        reference = "https://www.scottbudow.com/issues?id=f5b39b46-f3ca-4fc8-a546-9229dbfbd487"
+        defined_for = StateCode.NY
+
+        def formula(tax_unit, period, parameters):
+            income = tax_unit.household("household_net_income", period)
+            federal_poverty_guidelines = tax_unit("tax_unit_fpg", period)
+            children = tax_unit("tax_unit_children", period)
+            p = parameters(
+                period
+            ).gov.contrib.scott_budow.end_entrenched_poverty
+            fpg_ratio = income / federal_poverty_guidelines
+            children_amount = p.child.calc(fpg_ratio) * children
+            family_size = tax_unit("tax_unit_size", period)
+            base_amount = p.base.calc(fpg_ratio) * family_size
+            return children_amount + base_amount
+
+    class ny_refundable_credits(Variable):
```

Comment:
can you try to modify the list parameter instead?

Reply:
> can you try to modify the list parameter instead?

Hi Max, trying to solve this by adding a modify_parameters function, but fail to pass the tests. Pavel and I are not sure whether we can change the list like this, instead of changing a specific amount. Could you take a look at the updated code and give some advice?